06-files-input-and-output/csv_split.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) 2016 Ami . All rights reserved
+import argparse
+import csv
+import os
+import sys
+
+def get_arguments():
+    current_directory = os.getcwd()
+    parser = argparse.ArgumentParser(description="Split CSV by every x rows")
+
+    parser.add_argument("-i", action="store", dest="input_file", help="CSV input file")
+    parser.add_argument("-o", action="store", dest="output_path",
+                        help="Output path - where to store the split CSV files",
+                        default=current_directory)
+    parser.add_argument("-r", action="store", dest="row_limit",
+                        help="split by every x lines", type=int, required=True)
+    if len(sys.argv) == 1:
+        parser.print_help()
+        return False
+    else:
+        return parser.parse_args()
+
+def check_if_file_exists(file):
+    if os.path.exists(file):
+        return True
+    else:
+        print("File not found:", file)
+        return False
+
+def validate_minimum_rows(file, row_limit):
+    print("Validating file {} has more than {} rows to split by...".format(file, row_limit),
+        end=""
+    )
+    file_lines = sum(1 for line in open(file))
```

Comment:
Can you use the `_` argument in python3? it should be used instead of `line` since you don't do anything with `line`. And please extract this to a function named something like `get_lines_count`.